lib/speech/operation.js

```diff
@@ -0,0 +1,187 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module speech/operation
+ */
+
+'use strict';
+
+var events = require('events');
+var modelo = require('modelo');
+
+var OPERATION_FIELDS = [
+  'name',
+  'metadata',
+  'done',
+  'error',
+  'response'
+];
+
+function pick(dest, src) {
```

Comment:
Can you add a doc block to explain the purpose of this function? Also it should be a "private" function on `Operation` so it can be tested.